modules/system/models/File.php

```diff
@@ -42,13 +42,13 @@ public function getThumb($width, $height, $options = [])
     /**
      * {@inheritDoc}
      */
-    public function getPath()
+    public function getPath($fileName = null)
     {
         $url = '';
```

Comment:
@LukeTowers, minor note: could we return value and not save it to a local variable `$url`, that's not even needed)

Reply:
Nah, I usually prefer to initialize a default value to return instead of having multiple return statements. Not really a big deal here, but I don't see a reason to change it.